prow.sh

```diff
@@ -832,8 +832,19 @@ install_snapshot_controller() {
               echo "$modified"
               exit 1
           fi
-	  echo "kubectl apply -f ${SNAPSHOT_CONTROLLER_YAML}(modified)"
       done
+  elif [ "${CSI_PROW_DRIVER_CANARY}" = "canary" ]; then
```

Comment:
@pohly, should we also check if CSI_PROW_KUBERNETES_VERSION is the latest?
```
if [ "${CSI_PROW_KUBERNETES_VERSION}" = "latest" ] || [ "${CSI_PROW_DRIVER_CANARY}" = "canary" ]; then
```

Reply:
I see the following in the logs:
```
external-snapshotter version tag: CSI_SNAPSHOTTER_VERSION=master
using           image: k8s.gcr.io/sig-storage/csi-snapshotter:v3.0.3
```
csi-snapshotter:v3.0.3 is from `https://github.com/kubernetes-csi/csi-driver-host-path/blob/master/deploy/kubernetes-1.19`.
So we are testing csi-snapshotter:v3.0.3 with snapshot-controller:v4.2.0 (latest stable version).

Reply:
The question still is: do you want to test the latest stable snapshot-controller in the "on-kubernetes-master" job or the latest canary?